pkg/controllers/status/cluster_status_controller.go

```diff
@@ -94,6 +109,13 @@ func (c *ClusterStatusController) syncClusterStatus(cluster *v1alpha1.Cluster) (
 		return controllerruntime.Result{Requeue: true}, err
 	}
 
+	// register listers for pods and nodes in member cluster
+	clusterListerManager, err := c.registerListersAndStart(cluster)
+	if err != nil {
+		klog.Errorf("Failed to register informer for Cluster %s. Error: %v.", cluster.GetName(), err)
+		return controllerruntime.Result{Requeue: true}, err
+	}
+
```

Comment:
`registerListersAndStart` more like an initialization process, it not necessary to be called in each reconcile loop, even though nothing changed after the first call.

Reply:
The function's name may be not distinct. This function will start a informer if the informer manager does not exist, otherwise it will return the informer manager. I have changed the function name to `getOrCreateListers`.